site/src/github.rs

```diff
@@ -637,6 +637,32 @@ pub async fn post_finished(data: &InputData) {
                 Some(Direction::Regression | Direction::Mixed) => "+perf-regression",
                 Some(Direction::Improvement) | None => "-perf-regression",
             };
+            let msg = direction
+                .map(|d| {
+                    format!(
+                        "While you can manually mark this PR as fit \
+            for rollup, we strongly recommend not doing so since this PR led to changes in \
+            compiler perf.{}",
+                        match d {
+                            Direction::Regression | Direction::Mixed =>
+                                "\n\n**Next Steps**: If you can justify the \
+                regressions found in this perf run, please indicate this with \
+                `@rustbot label: +perf-regression-justified` along with \
+                sufficient written justification. If you cannot justify the regressions \
+                please fix the regressions and do another perf run. If the next run shows \
+                neutral or positive results, the label will be automatically removed.",
+                            _ => "",
```

Comment:
Minor point: I prefer exhaustive matching in case we change things up (though probably not *really* likely we add more directions...).